packages/contracts/foundry.toml

```diff
@@ -13,4 +13,10 @@ extra_output = [
 optimizer = true
 optimizer_runs = 20_000
 force = false
-# See more config options https://github.com/foundry-rs/foundry/tree/master/conf
\ No newline at end of file
+# See more config options https://github.com/foundry-rs/foundry/tree/master/conf
+[fmt]
+tab_width = 2
+line_length = 80
+tab_width = 4
+bracket_spacing = false
+qutoe_style = 'double'
```

Comment:
This is probably a typo. The Foundry link in the comment is broken. Can you verify this config schema?

Reply:
nice catch! yeah its a typo.

Reply:
https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/91ee852c97a0153196f46e0462eb019c9579cb04
https://github.com/ubiquity/ubiquity-dollar/pull/261/commits/850acc670e231331bc24b36d1ba2485553996995